include/eosio/vm/types.hpp

```diff
@@ -337,8 +337,10 @@ namespace eosio { namespace vm {
       template<typename Imports>
       static uint32_t get_imported_functions_size_impl(const Imports& imports) {
          uint32_t number_of_imports = 0;
-         for (uint32_t i = 0; i < imports.size(); i++) {
-            if (imports[i].kind == external_kind::Function)
+         const auto sz = imports.size();
+         const auto data = imports.data(); // to avoid the unnecessary size check since we iterate from 0 to sz
```

Comment:
Add a comment about why `imports[i]` is not used (the `at()` used by operator `[]` checks size).

Reply:
https://github.com/AntelopeIO/eos-vm/pull/28/commits/37ef889689654352d5cc52052f9c1f662db5f846